create_deposition/README.md

```diff
@@ -0,0 +1,42 @@
+# Action for Creating Zenodo-Depostion
+This action creates a deposition with the metadata provided in `metadata.json` file.
+
+## Usage
+To create a deposition,first add the metadata to `metadata.json` file in the root directory. 
+Run `ci/create_new_depostion.workflow`.
```

Comment:
Users need to define the action shown below in _their own_ workflows. That is, they do not need to run the  `ci/create_new_deposition.workflow` workflow themselves. This workflow file is defined by us so that Travis can run tests. So this line (L6) can be deleted.